src/main/resources/application-dev.yml

```diff
@@ -1,3 +1,13 @@
 spring:
   flyway:
     locations: classpath:/migrations/common,classpath:/migrations/dev
+
+notify:
+  templates:
+    probation:
+      cancelled-booking:
```

Comment:
Once this is tested in dev and the API has been pushed out we can switch over to the new templates in production.